1.8/administration/basic-guideline

```diff
@@ -0,0 +1,92 @@
+---
+layout: page
+title:  "First steps"
+categories: [administration]
+---
+
+## Administration panel
```

Comment:
The rest of the documentation refers to this as `Admin CP` or `Admin Control Panel`.

Reply:
Also other places that refer to the `administration panel` in the main body of the text should be changed to `Admin CP`.